exporter/clickhousetracesexporter/migrations/000020_attributes_mv.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS signoz_traces.tag_attributes ON CLUSTER cluster (
+    timestamp DateTime CODEC(ZSTD(1)), 
+    tagKey LowCardinality(String) CODEC(ZSTD(1)),
+    stringTagValue String CODEC(ZSTD(1)),
+    numberTagValue Nullable(Float64) CODEC(ZSTD(1)),
+    boolTagValue Nullable(bool) CODEC(ZSTD(1)),
+    INDEX idx_tagKey tagKey TYPE bloom_filter(0.01) GRANULARITY 64,
```

Comment:
we should to store tag type as well right ? like attribute or resource.

Reply:
Added.